ExponeaSDK/ExponeaSDKShared/Classes/Configuration.swift

```diff
@@ -107,7 +107,7 @@ public struct Configuration: Codable, Equatable {
         self.defaultProperties = defaultProperties
         self.sessionTimeout = sessionTimeout
         self.automaticSessionTracking = automaticSessionTracking
-        self.automaticPushNotificationTracking = automaticPushNotificationTracking
+        self.automaticPushNotificationTracking = true
```

Comment:
AutomaticPushNotificationTracking property is deprecated from version 2.7.0 and we decided to deprecate it because it uses method swizzling that causes issues when the host application uses multiple SDKs that do the same. Besides that, if we force set it to true, it would cause problems for other customers, who do not want to use swizzling. 

From docu:

AutomaticPushNotificationTracking is replaced by pushNotificationTracking. With pushNotificationTracking, you're more in control of what's happening inside your app and it also makes debugging easier. When migrating from automaticPushNotificationTracking some extra work is required. Please check Push notifications documentation for more details: https://github.com/exponea/exponea-ios-sdk/blob/develop/Documentation/PUSH.md